timeline-chart/src/components/time-graph-row-element.ts

```diff
@@ -34,7 +34,8 @@ export class TimeGraphRowElement extends TimeGraphComponent {
             x: this.range.start,
             y: this._row.position.y + ((this.row.height - this.height) / 2)
         };
-        const width = this.range.end - this.range.start;
+        // min width of a state should never be less than 1(for visibility)
+        const width = Math.max(1, this.range.end - this.range.start);
```

Comment:
I don't know how this is all used but:
What is range.start and range.end? Are they pixel x-coordinates? If start=4 and end=5, does it mean that state should be drawn from 4 to 5, and in that case does it mean that width should be 2? If all this is correct, I think that the formula should be width = end - start + 1 ?

Reply:
@PatrickTasse Yes, range.start and range.end are the x-coordinates adjusted with respect to view range of the chart. These values are from the server.

For your example, I am not sure if we have to set it to '1' or '2' pixels.  But I see for all width calculations, we have used 'end-start'.